src/getCoinCombination.test.js

```diff
@@ -3,8 +3,33 @@
 describe('getCoinCombination', () => {
   const { getCoinCombination } = require('./getCoinCombination');
 
-  it(`should be declared`, () => {
-    expect(getCoinCombination)
-      .toBeInstanceOf(Function);
+  it('should return an array of length 4', () => {
+    const result = getCoinCombination(1);
+
+    expect(result).toHaveLength(4);
+  });
+
+  it('should return the correct combination for 1 cent', () => {
+    const result = getCoinCombination(1);
+
+    expect(result).toEqual([1, 0, 0, 0]);
```

Comment:
lets make your checks a bit better)) Please, rewrite it using next approach:   it(`should return '1' pennie when '1' is entered`, () => {
    expect(getCoinCombination(1))
      .toEqual([1, 0, 0, 0]);
  });